node/cmd/guardiand/admintemplate.go

```diff
@@ -472,6 +519,115 @@ func runCircleIntegrationUpgradeContractImplementationTemplate(cmd *cobra.Comman
 	fmt.Print(string(b))
 }
 
+func runWormchainStoreCodeTemplate(cmd *cobra.Command, args []string) {
+	if *wormchainStoreCodeWasmHash == "" {
+		log.Fatal("--wasm-hash must be specified.")
+	}
+	wasmHash, err := parseAddress(*wormchainStoreCodeWasmHash)
+	if err != nil {
+		log.Fatal("failed to parse new implementation address: ", err)
+	}
```

Comment:
Why do we call `parseAddress` here? The hash should already be a 32-byte string, no need for padding. Calling `parseAddress` would technically allow a template to be generated with a `base58` or `bech32` encoded string. Not much danger since the VAA then needs to be signed, but can be a little misleading.

Can we use the same validation here that the `adminserver` does? Similar to
```
	// validate the length of the hex passed in
	b, err := hex.DecodeString(req.WasmHash)
	if err != nil {
		return nil, fmt.Errorf("invalid cosmwasm bytecode hash (expected hex): %w", err)
	}

	if len(b) != 32 {
		return nil, fmt.Errorf("invalid cosmwasm bytecode hash (expected 32 bytes but received %d bytes)", len(b))
	}

	wasmHash := [32]byte{}
	copy(wasmHash[:], b)
```

Reply:
Fixed.